[2021-12-11 16:54:23,818] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: dag_datalake_postgres.ingest_postgres_table 2021-12-11T16:53:39.337041+00:00 [queued]>
[2021-12-11 16:54:23,833] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: dag_datalake_postgres.ingest_postgres_table 2021-12-11T16:53:39.337041+00:00 [queued]>
[2021-12-11 16:54:23,834] {taskinstance.py:1087} INFO - 
--------------------------------------------------------------------------------
[2021-12-11 16:54:23,835] {taskinstance.py:1088} INFO - Starting attempt 2 of 2
[2021-12-11 16:54:23,835] {taskinstance.py:1089} INFO - 
--------------------------------------------------------------------------------
[2021-12-11 16:54:23,843] {taskinstance.py:1107} INFO - Executing <Task(PythonOperator): ingest_postgres_table> on 2021-12-11T16:53:39.337041+00:00
[2021-12-11 16:54:23,848] {standard_task_runner.py:52} INFO - Started process 438 to run task
[2021-12-11 16:54:23,852] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'dag_datalake_postgres', 'ingest_postgres_table', '2021-12-11T16:53:39.337041+00:00', '--job-id', '4', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/dag_datalake_postgres.py', '--cfg-path', '/tmp/tmp5lljtq38', '--error-file', '/tmp/tmp2omuml30']
[2021-12-11 16:54:23,853] {standard_task_runner.py:77} INFO - Job 4: Subtask ingest_postgres_table
[2021-12-11 16:54:23,899] {logging_mixin.py:104} INFO - Running <TaskInstance: dag_datalake_postgres.ingest_postgres_table 2021-12-11T16:53:39.337041+00:00 [running]> on host bc9f9503eb6e
[2021-12-11 16:54:23,946] {taskinstance.py:1302} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=dag_datalake_postgres
AIRFLOW_CTX_TASK_ID=ingest_postgres_table
AIRFLOW_CTX_EXECUTION_DATE=2021-12-11T16:53:39.337041+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2021-12-11T16:53:39.337041+00:00
[2021-12-11 16:54:23,998] {postgres_ingestion.py:28} INFO - Queried data:    id creation_date updated_date  company_id  cash_in  cash_out
0   1    2021-10-01   2021-10-01         234      450       455
1   2    2021-11-01   2021-11-01         340       40        55
2   3    2021-11-01   2021-11-01         340       50        55
3   4    2021-11-01   2021-11-03         365     1340      3455
4   5    2021-11-02   2021-11-04         365     1200      2455
5   6    2021-11-04   2021-11-04         450        0         4
6   7    2021-11-12   2021-11-12         450        0        55
7   8    2021-11-15   2021-11-20         450        0        45
[2021-12-11 16:54:24,081] {python.py:151} INFO - Done. Returned value was: None
[2021-12-11 16:54:24,096] {taskinstance.py:1211} INFO - Marking task as SUCCESS. dag_id=dag_datalake_postgres, task_id=ingest_postgres_table, execution_date=20211211T165339, start_date=20211211T165423, end_date=20211211T165424
[2021-12-11 16:54:24,177] {taskinstance.py:1265} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2021-12-11 16:54:24,266] {local_task_job.py:149} INFO - Task exited with return code 0
